proto/pocket/supplier/tx.proto

```diff
@@ -1,7 +1,18 @@
 syntax = "proto3";
+
 package pocket.supplier;
 
 option go_package = "pocket/x/supplier/types";
 
 // Msg defines the Msg service.
-service Msg {}
\ No newline at end of file
+service Msg {
+  rpc CreateClaim (MsgCreateClaim) returns (MsgCreateClaimResponse);
+}
+message MsgCreateClaim {
+  string supplierAddress = 1;
```

Comment:
We should decide on whether we use camel or snake case for our protos

Reply:
Thanks for catching this, it should be snake case.